File: psat_api_private.py
import http.client
import urllib
import requests
import json

req ={
    'method'          : 'GET', 
    'key'             : '•••••••••', 
    'uri'             : 'https://results.eu.securityeducation.com:443', 
    'command'         : '/api/reporting/v0.1.0/cyberstrength', 
    'parameters'      : ''
   }


headers    = {'x-apikey-token' : req['key']}
response   = requests.get(req['uri'] + req['command'] + req['parameters'], headers=headers)
res        = response.json()
print(res)

File: psat_api_private.js
const axios = require('axios'); 

const req ={
    method          : 'GET', 
    key             : '•••••••••', 
    uri             : 'https://results.eu.securityeducation.com:443', 
    command         : '/api/reporting/v0.1.0/cyberstrength', 
    parameters      : ''
  }

const params        = {headers: {'x-apikey-token': req.key }}

main(); // calling async function to make await possible
async function main () {
  let eti = await axios.get(req.uri+req.command+req.parameters, params       );
  console.log(eti); 
} 
